backend/functions/lib/configs/application-configuration.ts

```diff
@@ -80,6 +82,54 @@ export class ApplicationConfiguration {
     return config;
   }
 
+  // TODO: This and getDxtrDbConfig are gross. Refactor when we feel we really want to clean this up.
```

Comment:
If we are going to refactor these, we should probably make it into an intentional ticket. Or part of an immediately following PR